### PR TITLE
Constrained Max Height 

### LINKED ISSUE #1574: Limit Max Height
image

- Max height should be constrained to the height of the screen, should not have to scroll for the button

### PR DESCRIPTION
**Pull request by SamarthSrinivasa**
constrained max heigh to 85vh and width to 90% (arbitrary?), button shows at the bottom of the screen. 
Screenshot 2024-08-10 at 5 47 49 PM

### CODE REVIEW

File: src/components/admin/services/checkin/CheckIn.jsx [hunk 1]
```diff
@@ -81,16 +84,30 @@ const CheckIn = () => {
               userFn={(event) => setEvent(event)}
             />
           )}
-          <Scanner setResult={setResult} />
+          <div
+            className="flex w-full flex-col items-center justify-center"
+            style={{ maxHeight: "85vh", width: "90%", overflow: "hidden" }}
```
Comment (shahdivyank) [inline src/components/admin/services/checkin/CheckIn.jsx]: tailwind

File: src/components/admin/services/checkin/CheckIn.jsx [hunk 2]
```diff
@@ -70,8 +70,11 @@ const CheckIn = () => {
   return (
     <div className="flex h-full flex-col gap-3 py-4 font-poppins">
       <Title title="Check In" />
-      <div className="grid grid-cols-1">
-        <div className="flex flex-col items-center gap-3 p-3">
+      <div className="grid flex-grow grid-cols-1 overflow-auto">
+        <div
+          className="flex flex-col items-center gap-3 p-3"
+          style={{ flexGrow: 1, overflow: "hidden" }}
```
Comment (shahdivyank) [inline src/components/admin/services/checkin/CheckIn.jsx]: tailwind

File: src/components/admin/services/checkin/CheckIn.jsx [hunk 3]
```diff
@@ -81,16 +84,30 @@ const CheckIn = () => {
...
             />
           )}
-          <Scanner setResult={setResult} />
+          <div
+            className="flex w-full flex-col items-center justify-center"
+            style={{ maxHeight: "85vh", width: "90%", overflow: "hidden" }}
+          >
+            <Scanner
+              setResult={setResult}
+              style={{ height: "100%", width: "100%" }}
```
Comment (shahdivyank) [inline src/components/admin/services/checkin/CheckIn.jsx]: tailwind

File: src/components/admin/services/checkin/CheckIn.jsx [hunk 4]
```diff
@@ -81,16 +84,30 @@ const CheckIn = () => {
...
-            color="green"
-            onClick={handleCheckIn}
-            size="text-xl"
-          />
         </div>
       </div>
+
+      <div
+        className="flex flex-shrink-0 justify-center p-6"
+        style={{ paddingTop: "5px", paddingBottom: "10px" }}
```
Comment (shahdivyank) [inline src/components/admin/services/checkin/CheckIn.jsx]: tailwind

File: src/components/admin/services/checkin/CheckIn.jsx [hunk 5]
```diff
@@ -70,8 +70,11 @@ const CheckIn = () => {
   return (
     <div className="flex h-full flex-col gap-3 py-4 font-poppins">
       <Title title="Check In" />
-      <div className="grid grid-cols-1">
-        <div className="flex flex-col items-center gap-3 p-3">
+      <div className="grid grid-cols-1 overflow-auto">
+        <div
+          className="flex flex-col gap-3 py-3"
+          style={{ overflow: "hidden" }}
```
Comment (SafeDuck) [inline src/components/admin/services/checkin/CheckIn.jsx:76]: tailwind

**Pull request closed without merging**